components/brave_news/browser/resources/feed/Card.tsx

```diff
@@ -69,7 +73,14 @@ const HidableImage = ({
     src += `&target_size=${width}x${height}`
   }
 
-  return <img {...rest} src={src} ref={ref as any} onError={handleError} onLoad={handleLoad} />
+  return <img
+    {...rest}
+    src={delayLoad ? undefined : src}
+    ref={ref as any}
+    onError={handleError}
+    onLoad={handleLoad}
+    style={{ opacity: 0, transition: 'opacity 400ms ease-in-out', ...rest.style }}
```

Comment:
I think rather than setting this style like this we should have a styled component:

```
const BaseImg = styled.img`
  opacity: 0;
  transition: 'opacity, 400ms, ease-in-out'
`;
```

and use that instead of the `img` here.